backend/app/assets/stylesheets/spree/backend/components/_main.scss

```
@@ -3,11 +3,35 @@
   margin-top: 50px;
 }
 
-.sidebar-minimized #main-part {
-  padding: $spacer ($spacer * 2) 0 ($spacer * 5);
-}
-
 #logo {
   width: auto;
-  height: 40px;
+  height: 36px;
+}
+
+#multi-backdrop {
```

```suggestion
.modal-backdrop {
```
please, use the class instead of id